pkg/net/net_linux.go

```diff
@@ -66,15 +51,10 @@ func nics(ctx *context.Context) []*NIC {
 
 		mac := netDeviceMacAddress(paths, filename)
 		nic.MacAddress = mac
-		nic.MACAddress = mac
```

Comment:
Please add this back in. The non-deprecated field is `MACAddress`.

Reply:
fixed, thanks